src/definitions/modules/progress.js

```diff
@@ -56,8 +71,8 @@ $.fn.progress = function(parameters) {
         moduleNamespace = 'module-' + namespace,
 
         $module         = $(this),
-        $bar            = $(this).find(selector.bar),
-        $progress       = $(this).find(selector.progress),
+        $bars           = $(this).find(selector.bar),
+        $progresss      = $(this).find(selector.progress),
```

Comment:
It should read `progresses` I think. I first thought it's a typo because of the trailing triple s :☺

Reply:
Fixed typo in https://github.com/fomantic/Fomantic-UI/pull/582/commits/9bc0dad8377b8b14b83463f0f4cce9d8e6722772